### PR TITLE
Use lzma to decompress .tar.xz debian files.

### PR DESCRIPTION
**Pull request by hrfuller**
Recent debian packages from sid can come XZ compressed. This lets you use lzma to build these .debs if it is installed and gives a helpful error message if it isn't.

### CODE REVIEW

File: container/build_tar.py [hunk 1]
```diff
@@ -259,7 +272,16 @@ def add_deb(self, deb):
       while current:
         parts = current.filename.split(".")
         name = parts[0]
-        ext = '.'.join(parts[1:])
+        if parts[-1] == 'xz':
```
Comment (nlopezgi) [inline container/build_tar.py]: Could you try to do the import of lzma only if needed (i.e., inside this conditional, as in this PR: https://github.com/bazelbuild/rules_docker/pull/433/files)

**Comment by nlopezgi** [comment]
can you also please sign the CLA

File: container/build_tar.py [hunk 2]
```diff
@@ -271,7 +271,21 @@ def add_deb(self, deb):
       while current:
         parts = current.filename.split(".")
         name = parts[0]
-        ext = '.'.join(parts[1:])
+        if parts[-1].lower() == 'xz':
+          try:
```
Comment (nlopezgi) [inline container/build_tar.py]: any reason why you can't just reuse _xz_decompress here?

Comment (hrfuller) [reply container/build_tar.py]: No, that makes sense. Ill change it to support backports for python 2.7.

**Pull request merged**